frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -72,6 +71,32 @@ <h5 class="fw-light">Challenge Settings</h5>
       </div>
     </div>
 
+    <!-- Evaluations and Annoatations -->
+    <div class="row settings-section">
+      <div class="col-lg-3 col-md-4 col-sm-3 col-xs-3">
+        <span class="fw-light fs-18">Evaluation and Annoatations:</span>
+      </div>
+      <div class="ev-action">
+        <div *ngIf="isChallengeHost" class="col-sm-1.5 col-xs-1.5 ev-upload-action">
+          <!-- Evaluations Criteria Description -->
+          <span class="fs-16 fw-light">
+            <strong class="text-light-black">Evaluation Criteria </strong>
+          </span>
+          <a class="pointer fs-16" (click)="editEvaluationCriteria()">
+            <i class="fa fa-pencil" aria-hidden="true"> </i>
+          </a>
+          <br />
+          <!-- Evaluation Script -->
+          <span class="fs-16 fw-light">
+            <strong class="text-light-black">Evaluation Script </strong>
```

Comment:
You can show `Evaluation script` in column next to `Evaluation Criteria` in second row. Use same font for both

Reply:
So both of them should be on the same row?

Reply:
Yes, same row different columns. Otherwise, there is a lot of empty space on right side of the section. If you have better arrangement suggestions feel free to share.

Reply:
So, I will add as mentioned, both of them on the same row so that we can add all the features, and till then I will look for a better design.